client/src/call-customer/components/clock.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Bravo!
This the meaning of SOC. You extracted the functionality of the clock as a widget in the Application.